src/applications/ivc-champva/shared/components/applicantLists/ApplicantAddressPage.jsx

```diff
@@ -22,14 +23,17 @@ export function ApplicantAddressCopyPage({
   positivePrefix,
   negativePrefix,
 }) {
+  const addressKey = customAddressKey ?? 'applicantAddress';
```

Comment:
Added the `customAddressKey` prop and this new `addressKey` constant, and then updated all usage of `'applicantAddress'` to instead use `addressKey` so that we can maintain backward compatibility while supporting arbitrary address properties now.